app/stores/index.js

```diff
@@ -1,14 +1,16 @@
-import { browserHistory }               from 'react-router';
-import { createStore, applyMiddleware } from 'redux';
-import { routerMiddleware }             from 'react-router-redux';
-import thunkMiddleware                  from 'redux-thunk';
+import { browserHistory }                        from 'react-router';
+import { createStore, applyMiddleware, compose } from 'redux';
+import { routerMiddleware }                      from 'react-router-redux';
+import thunkMiddleware                           from 'redux-thunk';
 
 import { loadState, saveState } from '../utils/local-storage';
 import { reducer }              from '../reducers';
 
 const initialState = loadState();
 const router = routerMiddleware(browserHistory);
 
-export const Store = createStore(reducer, initialState, applyMiddleware(thunkMiddleware, router));
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
I'd rather not have the extension load in production, so could you change this to:

```js
const composeEnhancers = process.env.NODE_ENV === 'production' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```